tests/unit/limitations_test.c

```diff
@@ -401,6 +404,55 @@ CTEST2(limitations, test_disk_size_not_integral_multiple_of_extents)
    ASSERT_NOT_EQUAL(0, rc);
 }
 
+/*
+ * **************************************************************************
+ * Test that an invalid configuration that makes trunk node configuration
+ * impractical fails correctly with an error message. We try out few diff
+ * config params that go into error checks in trunk_config_init().
+ * **************************************************************************
+ */
+CTEST2(limitations, test_trunk_config_init_fails_for_invalid_configs)
```

Comment:
These are really not "limitations" but checks for invalid configs; i.e., negative test cases.

I don't have a very good place to dump these into, so for now ... this file should suffice.